plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.properties

```diff
@@ -3,4 +3,4 @@ providerName=Google, Inc.
 
 deployStandardCommandDescription=Deploy the project to App Engine Standard
```

Comment:
This is a tooltip, right? Ideally it should use different language than the menu item it describes. E.g. Upload the project to App Engine. If someone doesn't understand "Deploy to App Engine Standard" they're probably not going to understand "Deploy the project to App Engine Standard"

I.e. tooltips should not parrot the thing they're describing.